cloudify_marketplace/blueprint/scripts/configure_openstack.py

```diff
@@ -0,0 +1,376 @@
+from copy import copy
```

Comment:
Corrected the ones in the external imports. I think the stdlib ones are correct?